service/history/task/timer_active_task_executor.go

```diff
@@ -140,6 +145,10 @@ func (t *timerActiveTaskExecutor) executeUserTimerTimeoutTask(
 	referenceTime := t.shard.GetTimeSource().Now()
 	resurrectionCheckMinDelay := t.config.ResurrectionCheckMinDelay(mutableState.GetDomainEntry().GetInfo().Name)
 	updateMutableState := false
+	debugLog := t.logger.Debug
```

Comment:
Why not move it to the top?

Reply:
I didn't want to change the level of the debug log at the top. It can stay purely as debug level